app/Resources/views/base.html.twig

```diff
@@ -134,17 +134,17 @@
-    	                <li><b>Default Boolean Operator:</b> by default terms in a search string will be connected by the AND operator, EXAMPLE: searching genomic melanoma will produce the same results as genomic AND melanoma. </li> &nbsp;
+    	                <li><b>Default Boolean Operator:</b> Terms in a search string will be connected by the <b>AND</b> operator, EXAMPLE: searching <b>genomic melanoma</b> will produce the <b>same results as genomic AND melanoma</b>. </li> &nbsp;
 
-		                <li><b>AND, OR, NOT:</b> you can combine terms in a search with the boolean operators AND (to add specificity), OR (to broaden), or NOT (to eliminate a term), the operators must be capitalized (all other search terms are not case specific).</li> &nbsp;
+		                <li><b>AND, OR, NOT:</b> Combine terms in a search with the boolean operators <b>AND</b> (to add specificity), <b>OR</b> (to broaden), or <b>NOT</b> (to eliminate a term). Operators must be capitalized (all other search terms are not case-sensitive).</li> &nbsp;
 
-		                <li><b>Wild Card:</b> Add wild cards such as * (open-ended wildcard) and ? (single character) to truncate terms, EXAMPLE: searching gen* shows results for 'gene', 'genetic', 'genomic, etc., searching gene? returns results for 'gene' or 'genes' but not for 'genome'. </li> &nbsp;
+		                <li><b>Wild Card:</b> Add wild card symbols <b>*</b> (open-ended) and <b>?</b> (single character) to truncate terms. EXAMPLE: searching <b>gen*</b> shows results for 'gene', 'genetic', 'genomic, etc. Searching <b>gene?</b> returns results for 'gene' or 'genes' but not for 'genome'. </li> &nbsp;
 
-		                <li><b>Phrase Searching:</b> encapulate your search in quotation marks "" to search for an exact phrase, EXAMPLE: "transcription factors" returns results for cases when transcription is immediately followed by factors. </li> &nbsp;
+		                <li><b>Phrase Searching:</b> Enclose search in quotation marks <b>"keyword or search term"</b> to search for an exact phrase. EXAMPLE: <b>"transcription factors"</b> returns results for records where <b>transcription</b> is immediately followed by the word <b>factors</b>. </li> &nbsp;
 
-		                <li><b>Diacritics:</b> Words containing diacritics translate those characters to non-diacritic form, EXAMPLE: massague returns results for Massague&#769. </li> &nbsp;
+		                <li><b>Diacritics:</b> Words containing diacritics translate those characters to non-diacritic form. EXAMPLE: <b>massague</b> returns results for <b>Massague&#769</b>. </li> &nbsp;
 
-		                <li><b>Filters:</b> in left panel, remove filter by clicking on the term at the top of the page. </li> &nbsp;
+		                <li><b>Filters by:</b> Located on the homepage (left panel), use filters to search for records or narrow the search. From search results page, filters can be removed by clicking on the term at the top of the page. </li> &nbsp;
```

Comment:
The only thing I see - "Filters by" should be "Filter by"

Reply:
Good catch! Thank you!